google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineDeploy.java

```diff
@@ -198,37 +145,29 @@ public void exit(int exitCode) {
 
           if (deployOutput == null
               || deployOutput.getService() == null || deployOutput.getVersion() == null) {
-            consoleLogLn(
-                GctBundle.message("appengine.deployment.version.extract.failure") + "\n");
+            loggingHandler.print(
```

Comment:
Part of the reason was to reuse a long error message in several places (a compromise we took), but I guess this is no longer the case here. In this case `appengine.action.error.update.message` is particularly unnecessary. So, I think we can log a plain string here.